src/libmongoc/tests/test-mongoc-sample-commands.c

```diff
@@ -4333,11 +4331,13 @@ test_sample_commands (void)
    test_sample_command (test_example_53, 53, db, collection, false);
    test_sample_command (test_example_54, 54, db, collection, true);
    test_sample_command (test_example_55, 55, db, collection, false);
+   test_sample_command (test_example_56, 56, db, collection, true);
```

Comment:
The `/Samples` tests are currently [being skipped](https://github.com/mongodb/mongo-c-driver/blob/0e54f99dada9db1196f82a7ef868f58ca8b0f022/.evergreen/etc/skip-tests.txt#L39). Manually unskipping and running the test suggests there is a test order dependency here: may need to move example 56 back below example 58.

Aside: `test_sample_txn_commands ()` also fails locally with a "shouldn't have logged" error which I believe is due to https://github.com/mongodb/mongo-c-driver/commit/36c1f141b55272f8d09a8492ef74fd9d701d16f6#diff-b71f20ac0b55a16f43592d6bc46791721a0a075febda9e508f3c66d555fae9e9, which I think can be addressed by replacing `ASSERT_NO_CAPTURED_LOGS()` with `ASSERT()` + `has_captured_log()` instead (for both warning and error levels?).